tools/triage/add_triage_signature.py

```diff
@@ -1323,6 +1324,78 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket(report)
 
 
+class IpChangedAfterReboot(ErrorSignature):
```

Comment:
is this problematic in any way that the IP changed after the reboot?
if so, in what way?

Reply:
IP address change is unexpected.  The nodes are identified by their IPs. Certificates are generated based on the IPs discovered during discovery phase.  One of the things that might happen is that one node can configure one of the VIPs to be its local IP.
In the case I worked on, crio tried to bind one of the IPs but it couldn't because the configured IP returned to the discovered IP.  So crio failed to start.

Reply:
Thanks for the explanation!